libxft will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libxft it will fail in the following way:

```
2024/11/12 23:12:45 INFO running step "test/pkgconf"
2024/11/12 23:12:45 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:12:45 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:12:45 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:12:45 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:12:45 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:45 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:45 WARN + basename /home/build/melange-out/libxft-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:45 WARN + dev_pkg=libxft-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:45 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:45 WARN + apk info -L libxft-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:45 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:45 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:45 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:45 WARN + grep -q ^Name: usr/lib/pkgconfig/xft.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:45 WARN + basename usr/lib/pkgconfig/xft.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:45 WARN + lib_name=xft uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:45 WARN + echo usr/lib/pkgconfig/xft.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:45 WARN + grep -q '^usr/lib/pkgconfig/xft.pc$\|^usr/share/pkgconfig/xft.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:45 WARN + pkgconf --exists xft uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:45 WARN + pkgconf --print-errors --exists xft uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:45 WARN Package xproto was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:45 WARN Perhaps you should add the directory containing `xproto.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:45 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:45 WARN Package 'xproto', required by 'xft', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:45 INFO pod 163ae8f9306e559ecba129ffbda00aaf906609728473474fab9e9dc6d0f363ed terminated
2024/11/12 23:12:45 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:12:45 INFO   workspace dir: /home/user/tmp/melange-workspace-566544358
2024/11/12 23:12:45 INFO   guest dir: /home/user/tmp/melange-guest-1631001616
2024/11/12 23:12:45 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libxft] Error 1
```

--- a/libxft.yaml
+++ b/libxft.yaml
@@ -1,7 +1,7 @@
 package:
   name: libxft
   version: 2.3.8
-  epoch: 1
+  epoch: 2
   description: FreeType-based font drawing library for X
   copyright:
     - license: MIT
@@ -48,8 +48,12 @@ subpackages:
     dependencies:
       runtime:
         - libxft
+        - xorgproto
         - zlib-dev
     description: libxft dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
   - name: libxft-doc
     pipeline:

